src/styles/theme.js

```diff
@@ -0,0 +1,64 @@
+// @ts-nocheck
+import { ThemeProvider, Global, css } from '@emotion/react'
+import PropTypes from 'prop-types'
+import normalize from 'emotion-normalize'
+import colors from './colors'
+import { breakpoints } from './breakpoints'
+
+const theme = {
+  colors,
+  breakpoints,
+}
+
+const Theme = ({ children }) => (
+  <ThemeProvider theme={theme}>
+    
+    
+    <Global
+      styles={css`
+      ${normalize}
+
+      :root {
+        --primary: ${({ theme }) => theme.colors.blue08};
+        --secondary: ${({ theme }) => theme.colors.yellow06};
+        --black: ${({ theme }) => theme.colors.gray09};
+        --light-gray: ${({ theme }) => theme.colors.gray02};
+        --light-gray-2: ${({ theme }) => theme.colors.gray04};
+        --white: white;
+        --danger: ${({ theme }) => theme.colors.gray09};
+        --blue-text: ${({ theme }) => theme.colors.blue06};
```

Comment:
I check the issue with the themes and I solve it by removing the arrow function and left only the variable.

![imagen](https://user-images.githubusercontent.com/66505715/155772095-9cab438f-ca4a-48d6-8ec6-95693d627d79.png)

I first do a console.log

`console.log(--primary: ${({ theme }) => theme.colors.blue08})` it and it returns to me the function instead of the return. I think that emotion maybe is receiving the function but not call it.

Changing it to just the variable works for me:

I meant:

```suggestion
        --primary: ${theme.colors.blue08};
        --secondary: ${theme.colors.yellow06};
        --black: ${theme.colors.gray09};
        --light-gray: ${theme.colors.gray02};
        --light-gray-2: ${theme.colors.gray04};
        --white: white;
        --danger: ${theme.colors.gray09};
        --blue-text: ${theme.colors.blue06};
```